### DISCUSSION TITLE
Backup/Restore

### DISCUSSION THREAD

**Discussion by woaloo** [leonlatsch/Photok#237] [General]
I have 2 smartphones with  Photok.
If I make a backup on one phone and want to restore on the other one, it's always fails.
I've tried with a 3 Gb backup , Photok crashed
I've tried with a 2Gb Backup, it never ending.
Is there a file size limit, or it's impossible ?

**Comment by leonlatsch** [comment]
No, there is so size limit. Its just that the backups are some times a little janky.
Ive heard this kind of issue before, but was not able to reproduce it by myself. It seems like some specific pictures cause this. Just like other crashes when loading. See #147 

If someone can reproduce this with a stacktrace, please open a bug issue. Thx.

For your issue, I would suggest exporting and creating backups yourself till this is fixed.
